lib/index.js

```diff
@@ -211,7 +218,7 @@ api.createSignatureString = ({includeHeaders, requestOptions} = {}) => {
   const {host, path} = _parseHostAndPath(url);
   return _signingString(
     {includeHeaders, headers, method, url, host, path,
-      validate: _validateSendHeader});
+      requestOptions, validate: _validateSendHeader});
```

Comment:
another unusual line break, maybe move open braces to 219, enumerate all the params as many as possible per line and close on a new line.

Reply:
Addressed in https://github.com/digitalbazaar/http-signature-header/pull/12/commits/1f18f7e132e46beba7bcca9f7eeca115e1bb90d4